doc/user-manual/language/cubical.lagda.rst

```diff
@@ -3,79 +3,117 @@
+    renaming ( primIMin to _∧_
+             ; primIMax to _∨_
+             ; primINeg to ~_
+             ; primHComp to hcomp
+             ; primTransp to transp
+             ; itIsOne to 1=1 )
+  open import Agda.Builtin.Cubical.Path
+  open import Agda.Builtin.Cubical.Sub
+    renaming ( primSubOut to ouc )
+  open import Agda.Builtin.Cubical.Glue public
+    using ( isEquiv
+          ; equiv-proof
+          ; _≃_
+          ; primGlue )
+  open import Agda.Builtin.Sigma public
 
-  open import Agda.Builtin.Cubical.Path renaming (_≡_ to Path)
+  infix 2 Σ-syntax
+
+  Σ-syntax : ∀ {ℓ ℓ'} (A : Set ℓ) (B : A → Set ℓ') → Set (ℓ ⊔ ℓ')
+  Σ-syntax = Σ
+
+  syntax Σ-syntax A (λ x → B) = Σ[ x ∈ A ] B
+
+  _×_ : ∀ {ℓ ℓ'} (A : Set ℓ) (B : Set ℓ') → Set (ℓ ⊔ ℓ')
+  A × B = Σ A (λ _ → B)
+
+  infixr 5 _×_
 
 .. _cubical:
 
 ***************************
 Cubical Type Theory in Agda
 ***************************
 
-Cubical Type Theory `Cohen et al., Cubical`_ is implemented in Agda (beta version).
+As of version 2.6.0 Agda has a cubical mode which extends Agda with a
+variety of features from Cubical Type Theory. In particular,
+computational univalence and higher inductive types which hence gives
+computational meaning to `Homotopy Type Theory and Univalent
+Foundations <https://homotopytypetheory.org/>`_. The version of
+Cubical Type Theory that Agda implements is a variation of the `CCHM`_
+Cubical Type Theory where the Kan composition operations are
+decomposed into homogeneous composition and generalized
+transport. This is what makes the general schema for higher inductive
+types work, following the `CHM`_ paper.
 
-To use cubical type theory, you need to run Agda with the ``--cubical`` command-line-option.
-You can also write ``{-# OPTIONS --cubical #-}`` at the top of the file.
+To use the cubical mode Agda needs to be run with the ``--cubical``
+command-line-option or with ``{-# OPTIONS --cubical #-}`` at the top
+of the file.
 
-To define paths, use λ abstractions. For example, this is the definition of the constant path:
+The cubical mode adds the following features to Agda:
 
-..
-  ::
-  module refl-example where
+1. An interval ``I`` type and path types
+2. Partial elements and systems
+3. Kan operations (``transp`` and ``hcomp``)
+4. ``Glue`` types
+5. Higher inductive types
+6. Cubical identity types
 
-::
+There is a standard library for Cubical Agda available at
+https://github.com/agda/cubical.
 
-    refl : ∀ {a} {A : Set a} {x : A} → Path x x
-    refl {x = x} = λ i → x
+The main design choices of the core part of the library are explained
+in https://homotopytypetheory.org/2018/12/06/cubical-agda/.
 
-Although they use the same syntax, a path is not a function.
-For example, the following is not valid:
+In order to get access to the Cubical Agda primitives one should
+either import
+https://github.com/agda/cubical/blob/master/Cubical/Core/Primitives.agda
+or add the relevant import statements from the top of that file.
 
```

Comment:
It would be good to have a code block indicating exactly how to start a file (the `{-# OPTIONS --cubical #-}` and the imports to make). It should be clearly visible as soon as the user opens the doc on cubical Agda, so that people immediately see what they need in order to start experimenting.

Maybe there should be several different ones depending on whether you are using [agda/cubical](https://github.com/agda/cubical), the HoTT-UF version of it, or simply the primitives included in Agda (maybe with suggested renamings), but then it should be made very clear which is which.

Reply:
I added the recommended one in the beginning now (using agda/cubical) and some pointer for experts. I also added instructions for how to use the HoTT-UF one in the section